### PR TITLE
Adding provision-with explanation on utilities page

### PR DESCRIPTION
**Pull request by Mte90**
Tiny explanation on how to use `provision-with` with utilities

### CODE REVIEW

File: docs/en-US/utilities.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It is possible to run a provision of an utility without run the whole VVV provision using a Vagrant command.  
+As example from the config.yml above `vagrant reload --provision-with=utility-core-php56`, where we are specifying the utility repo and the utility name.
```
Comment (tomjn) [inline docs/en-US/utilities.md]:
```suggestion
As an example from the `config.yml` above we can run `vagrant provision --provision-with="utility-core-php56"` to provision only the PHP 5.6 utility.
```

File: docs/en-US/utilities.md [hunk 2]
```diff
@@ -70,6 +70,12 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It is possible to run a provision of an utility without run the whole VVV provision using a Vagrant command.
```
Comment (tomjn) [inline docs/en-US/utilities.md]:
```suggestion
It's possible to provision a single utility on its own, without provisioning the whole of VVV using the `--provision-with` parameter of vagrant.  
```

File: docs/en-US/utilities.md [hunk 3]
```diff
@@ -70,6 +70,12 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It is possible to run a provision of an utility without run the whole VVV provision using a Vagrant command.  
+As example from the config.yml above `vagrant reload --provision-with=utility-core-php56`, where we are specifying the utility repo and the utility name.  
+This is helpful during the development or just to install/update a specific utility.
```
Comment (tomjn) [inline docs/en-US/utilities.md]:
```suggestion
```

File: docs/en-US/utilities.md [hunk 4]
```diff
@@ -70,6 +70,11 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It's possible to provision a single utility on its own, without provisioning the whole of VVV using the `--provision-with` parameter of vagrant.  
+As an example from the `config.yml` above we can run `vagrant provision --provision-with="utility-core-php56"` to provision only the PHP 5.6 utility.
```
Comment (msaggiorato) [inline docs/en-US/utilities.md]: One thing. This won't work if we don't push the config.yml file.

Eg: when needing to install a new PHP version.

Comment (Mte90) [reply docs/en-US/utilities.md]: Do you mean that we need to specify that before to execute that command is required to update the config file?

Comment (tomjn) [reply docs/en-US/utilities.md]: if you don't add PHP 5.6 as a utility to install in `config.yml` then this command won't work. There is no provisioner by that name in that scenario.

Comment (Mte90) [reply docs/en-US/utilities.md]: well we are talking about the example f of the config file in that page, so I don't know if we need to add something in the doc.

File: docs/en-US/utilities.md [hunk 5]
```diff
@@ -70,6 +70,12 @@ With this in mind, I would expect the java repository mentioned earlier to have
 
 The name of the subfolder maps directly on to what is put in `vvv-custom.yml`. VVV will run the `provision.sh` file, at which point it can do as it pleases. This could be installing a package via `apt-get` or something else. Other files can be included in these folders for `provision.sh` to make use of.
 
+## Utility Provisioning
+
+It's possible to provision a single utility on its own, without provisioning the whole of VVV using the `--provision-with` parameter of vagrant.  
+As an example from the `config.yml` above we can run `vagrant provision --provision-with="utility-core-php56"` to provision only the PHP 5.6 utility.  
+To work this command require that the utility is enabled in the `config.yml` file, it won't add it automatically.
```
Comment (tomjn) [inline docs/en-US/utilities.md]:
```suggestion
To work this command requires that the utility is enabled in the `config.yml` file, it won't add it automatically.
```

**Review by tomjn — APPROVED** [review]

**Pull request merged**